ibm-mq-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,80 @@
+{
+  new(this):
+    {
+      groups+: [
+        {
+          name: 'ibm-mq-alerts',
+          rules: [
+            {
+              alert: 'IBMMQExpiredMessages',
+              expr: |||
+                sum without (description,hostname,instance,job,platform) (ibmmq_qmgr_expired_message_count{%(filteringSelector)s}) > %(alertsExpiredMessages)s
+              ||| % this.config,
+              'for': '5m',
+              labels: {
+                severity: 'critical',
+              },
+              annotations: {
+                summary: 'There are expired messages, which imply that application resilience is failing.',
+                description:
+                  (
+                    'The number of expired messages in the {{$labels.qmgr}} is {{$labels.value}} which is above the threshold of %(alertsExpiredMessages)s.'
```

Comment:
Shouldn't the description reference the `$value` of metric? Probably need to format it as well: `{{ printf "%.0f" $value }}`, WDYT?